CONTRIBUTING.md

```diff
@@ -1,10 +1,3 @@
-+++
-title = "Contribuindo para o site"
-date = "2017-04-14T23:40:00-07:00"
-tags = ["osprogramadores"]
-categories = ["git"]
-banner = "img/banners/octocat.png"
-+++
 
```

Comment:
Extra blank line (remove).

Reply:
Extra blank line removed as requested. Thank you for the review. Please check.